File: src/main/java/core/constants/player/PlayerAggressiveness.java
package core.constants.player;

import core.model.TranslationFacility;

public class PlayerAggressiveness {
	private static final String[] languageKeys = {
			"ls.player.aggressiveness.tranquil",
			"ls.player.aggressiveness.calm",
			"ls.player.aggressiveness.balanced",
			"ls.player.aggressiveness.temperamental",
			"ls.player.aggressiveness.fiery",
			"ls.player.aggressiveness.unstable"};

	public static final int TRANQUIL = 0;
	public static final int CALM = 1;
	public static final int BALANCED = 2;
	public static final int TEMPERAMENTAL = 3;
	public static final int FIERY = 4;
	public static final int UNSTABLE = 5;


	private PlayerAggressiveness() {
	}

	public static String toString(Integer aggressiveness) {
		if (aggressiveness != null && aggressiveness >= TRANQUIL && aggressiveness <= UNSTABLE)
			return TranslationFacility.tr(languageKeys[aggressiveness]);
		return TranslationFacility.tr("Unbestimmt");
	}
}

File: src/main/java/core/constants/player/PlayerAggressiveness.kt
package core.constants.player

import core.model.HOVerwaltung

object PlayerAggressiveness {
    private val languageKeys = arrayOf(
        "ls.player.aggressiveness.tranquil",
        "ls.player.aggressiveness.calm",
        "ls.player.aggressiveness.balanced",
        "ls.player.aggressiveness.temperamental",
        "ls.player.aggressiveness.fiery",
        "ls.player.aggressiveness.unstable"
    )
    const val TRANQUIL = 0
    const val CALM = 1
    const val BALANCED = 2
    const val TEMPERAMENTAL = 3
    const val FIERY = 4
    const val UNSTABLE = 5
    @JvmStatic
    fun toString(aggressiveness: Int): String {
        return if (aggressiveness in TRANQUIL..UNSTABLE)
            HOVerwaltung.instance().getLanguageString(languageKeys[aggressiveness])
        else HOVerwaltung.instance().getLanguageString("Unbestimmt")
    }
}
